lib/card.rb

```diff
@@ -38,4 +38,25 @@ class Card < ActiveRecord::Base
   TRACKED_FIELDS = %w(name type_id db_content trash)
   Loader.load_mods if count > 0
 
+
+  def self.find_each(options = {})
+    if block_given?
+      find_in_batches(options) do |records|
+        records.each { |record| yield record }
+      end
+    else
+      super(options)
+    end
+  end
```

Comment:
And this is a copy of find_each in AR?

Seems like it should call the right find unless the code has an explicit class, right?  I mean, if this exact code is running un the AR::Base class, shouldn't self be the Card class when then executes, and the find_in_batches here on line 44 should be Card.find_in_batches.

Or I' missing something.  Presumably, this didn't work as expected until you did this, right?